app/views/freelancers/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Browse our Techies</h1>
+
+<% @freelancers.each do |freelancer| %>
+<div>
+  <%= image_tag(freelancer.image, size: "100x100", class: "image-fl-index", id: "image_#{freelancer.id}") %>
```

Comment:
I like that a class and id is included here. The id should make targeting this specific image easier to do in tests.

Reply:
thanks!